api_client/python/timesketch_api_client/search.py

```diff
@@ -573,6 +574,28 @@ def description(self, description):
         self._description = description
         self.commit()
 
+    @property
+    def expected_size(self):
+        """Property that returns the expected size of the search query."""
+        if self._total_elastic_size:
+            return self._total_elastic_size
+
+        # To get the total size of the query we first need to send a query
+        # so to minimize the cost we reduce the size to 10 entries.
+        max_entries = self._max_entries
+        scrolling = self.scrolling
+
+        self._max_entries = 10
```

Comment:
There is a better way of doing this. You can pass `count=True` to the search() call and just get back the count of matching documents. This is much cheaper as it doesn't have to retrieve anything.

Ref: https://github.com/google/timesketch/blob/master/timesketch/lib/datastores/elastic.py#L344

Reply:
I was attempting to remove the need to make changes in the actual API... made the changes and am now using the `count`